include/grpc/grpc_security.h

```diff
@@ -295,6 +295,33 @@ typedef struct grpc_server_credentials grpc_server_credentials;
    */
 GRPCAPI void grpc_server_credentials_release(grpc_server_credentials *creds);
 
+/** Server certificate store object holds the server's public certificates and
+ * associated private keys, as well as any CA certificates needed for client
+ * certificate validation (if applicable). */
+typedef struct grpc_ssl_server_certificate_store
```

Comment:
Sorry, maybe I wasn't clear we talked about this: for me the `store` is the callback + state. The actual keys and certs are the `grpc_ssl_server_credentials_config`.